src/server/network/network.ts

```diff
@@ -10,6 +11,7 @@ export type TeamData = {
 
 type Listener = {
   server: dgram.Socket;
+  ipv6: boolean;
```

Comment:
Do we actually need to know this after we have created it?

Reply:
It simplifies the logic for determining broadcast vs multicast